pkgs/applications/misc/prusa-slicer/super-slicer.nix

```diff
@@ -10,17 +10,19 @@ let
     src = fetchFromGitHub {
       owner = "supermerill";
       repo = "SuperSlicer";
-      sha256 = "sha256-Gg+LT1YKyUGNJE9XvWE1LSlIQ6Vq5GfVBTUw/A7Qx7E=";
+      sha256 = "sha256-W+0lbwWoaqaKqUWyFdETi6F/5dMfad+aUhNohG7O+Mg=";
       rev = version;
       fetchSubmodules = true;
     };
 
     # We don't need PS overrides anymore, and gcode-viewer is embedded in the binary.
     postInstall = null;
+    dontStrip = true;
```

Comment:
why is stripping preferable?

If you want to make "easy" to enable debugging, I would just create an option to enable it. But we shouldn't really be distributing debug builds for official releases

Reply:
Several (a lot, if ag is to be believed) other packages are already come with disabled stripping. I would guess that people are okay merging such stuff.

Also, an option would mean that you will recompile super-slicer yourself, and that's unbearable on print netbooks you usually use this software from — it takes 10 minutes even on a ±powerful machine.

Reply:
Well, I guess I should have focused more on the build being of type debug, which usually means without optimizations.

But stripping is usually preferred because it:
- Reduces binary size
- Removes potentially unneeded runtime dependencies (which contributes to closure bloat)

Reply:
> Reduces binary size

Depending on the type of binary this can be multiple times the final size of the binary.

Reply:
As a middle ground you can enable `separateDebugInfo = true;`

Reply:
separateDebugInfo = true is incompatible with dontStrip: the separatedebuginfo is created during stripPhase iirc